tests/test_vxlan_vtep.rb

```diff
@@ -30,6 +30,9 @@ def setup
   end
 
   def teardown
+    Feature.nv_overlay_disable
```

Comment:
n56-136 is in a failed state - will need a reboot. This is my run on my 5k:

ruby tests/test_vxlan_vtep.rb -e n6k -v
Run options: -e n6k -v --seed 5065

# Running:


Node under test:
  - name  - n6k-3
  - type  - N6K-C6001-72UP
  - image - bootflash:///aneesh-s.bin

TestVxlanVtep#test_shutdown = 17.09 s = .
TestVxlanVtep#test_create_negative = 16.29 s = .
TestVxlanVtep#test_source_interface_hold_down_time = 15.67 s = .
TestVxlanVtep#test_mt_full_create_destroy_multiple = 2.68 s = S
TestVxlanVtep#test_source_interface = 22.05 s = .
TestVxlanVtep#test_description = 17.71 s = .
TestVxlanVtep#test_create_destroy_one = 17.18 s = .
TestVxlanVtep#test_host_reachability = 17.62 s = .

Finished in 128.535827s, 0.0622 runs/s, 0.1945 assertions/s.

  1) Skipped:
TestVxlanVtep#test_mt_full_create_destroy_multiple [tests/test_vxlan_vtep.rb:86]: